react/features/subtitles/components/ClosedCaptionButton.web.js

```diff
@@ -6,7 +6,7 @@ import { translate } from '../../base/i18n/index';
 
 import { ToolbarButton } from '../../toolbox/';
 
-import { dialTranscriber, stopTranscribing } from '../actions';
```

Comment:
Is `dialTranscriber` still used? If not, should the action also be cleaned up? How does the transcriber work if it doesn't get dialed?

Reply:
no, it's not used anymore. By setting a local property (jitsi_participant_requestingTranscription=true), jicofo recognises that Jigasi needs to be invited.

Reply:
The whole "transcribing" feature currently lies unused, so I don't know what to do with it